### DISCUSSION TITLE
PROD vs DEV Enviorment

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#765] [Support / Q&A] [answered]
Is there a way to have a development project and then "Publish" changes into a production project within a workspace?

**Comment by harinij** [comment, marked as answer]
Hi @st-trade currently this is not supported directly, we are working on it, and will be coming soon. In the meantime, you have options to export/import table settings for configs of Rowy tables. As well as csv import/export for data.

**Reply by st-trade** [reply]
Okay, I'll try that manual config style, thanks
